app/controllers/rages_controller.rb

```diff
@@ -2,6 +2,8 @@ class RagesController < ApplicationController
   before_action :set_rage, only: [:show, :edit, :update, :destroy]
   before_filter :check_user_logged_in!, :except => [:show, :index]
 
+require 'uri'
```

Comment:
I did'nt work on that part, but perhaps all the linked stuff so should be in the controller and not only the require, to be clearer and logic-based.
However the usage is simple in the view files, see the ones where are used for instance the basename function that requires uri methods.